server/chat-plugins/help.ts

```diff
@@ -15,18 +15,16 @@ const PATH = 'config/chat-plugins/help.json';
 // 6: filters out conveniently short aliases
 const MINIMUM_LENGTH = 6;
 
-export let helpData: PluginData;
+export let helpData: PluginData = {
+	stats: {},
+	pairs: {},
+	disabled: false,
+	queue: [],
+};
 
 try {
-	helpData = JSON.parse(FS(PATH).readIfExistsSync() || "{}");
-} catch (e) {
-	helpData = {
-		stats: {},
-		pairs: {},
-		disabled: false,
-		queue: [],
-	};
-}
+	helpData = JSON.parse(FS(PATH).readIfExistsSync());
+} catch (e) {}
```

Comment:
I'm not sure the catch is correct, either. The only error we want to suppress is "doesn't exist". We should actually just use a regular `read`, and re-throw if we get any error other than "file not found".

Reply:
We can even initialize to your initial state there.